src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -668,6 +646,20 @@ private string MapServerEdition(ReliableConnectionHelper.ServerInfo serverInfo)
             return serverEdition;
         }
 
+        private static void FillInDefaultDetailsForConnections(ConnectionDetails ConnectionDetails, string featureName) { 
+
+            if(string.IsNullOrEmpty(ConnectionDetails.ApplicationName)) {
+                ConnectionDetails.ApplicationName = DefaultApplicationName;
+            }
+            else {

```

Comment:
General comment to be applied everywhere: try to follow the conventions adopted (at least) in this files, which is to align matching curly braces under the same column.